File: eva-core/eva-core-common/src/main/java/org/pkaq/core/annotation/Ignore.java
package org.pkaq.core.annotation;

import java.lang.annotation.*;

/**
 * 忽略数据权限注解
 * @author PKAQ
 */
@Target({ElementType.METHOD, ElementType.TYPE})
@Retention(RetentionPolicy.RUNTIME)
@Documented
public @interface Ignore {

}

File: eva-core/src/main/kotlin/io/nerv/core/annotation/Ignore.kt
package io.nerv.core.annotation

import java.lang.annotation.Documented
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

/**
 * 忽略数据权限注解
 */
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER, AnnotationTarget.ANNOTATION_CLASS, AnnotationTarget.CLASS)
@Retention(RetentionPolicy.RUNTIME)
@Documented
annotation class Ignore 